deploy.sh

```diff
@@ -186,21 +191,23 @@ install_specs() {
     populate_mirror "${what}"
 
     log "gathering specs"
-    spec_list="$(filter_specs ${HOME}/specs.txt)"
+    spec_list=$(spack filter --not-installed $(cat ${HOME}/specs.txt))
```

Comment:
Instead of `$(cat`, I think it's better to use the built-in `$(<`.
```suggestion
    spec_list=$(spack filter --not-installed $(< ${HOME}/specs.txt))
```